### ISSUE TITLE
How to improve the processing

### ISSUE THREAD

**Issue by whitetiger21022014** [nicholaskajoh/ivy#43]
Thank for your code
But it's so slow
I see the document as below
"Real-time and video processing object detection using Tensorflow, OpenCV and Docker.
https://towardsdatascience.com/real-time-and-video-processing-object-detection-using-tensorflow-opencv-and-docker-2be1694726e5 "

Do you think if we use Docker as that document the speed will be improved or not?
Thank you

**Comment by nicholaskajoh** [comment]
No, Docker won't improve the processing speed.

To get better performance:
- Use a GPU. You can currently do this with either the `tfoda` (TensorFlow) or `detectron2` (PyTorch) detector.
- Train a light-weight model. The model used in the demo is quite large and most of it is not being utilized.
- Use input video of lower framerate/resolution. This might negatively impact accuracy though.
